### ISSUE TITLE
after finetuning mt5 by transformers，generate effect become strange

### ISSUE THREAD

**Issue by yayaQAQ** [google-research/multilingual-t5#45]
I try to do translation task by mt5, exactly English to Chinese.
When I finetuning many steps on millions of parallel corporas, loss was down to 2.407 and stayed for a while. I try use the model after finetuning.
loss pic:
![image](https://user-images.githubusercontent.com/20497531/103390248-65a86880-4b4e-11eb-9d75-2ad1fd5dd50c.png)

My input is English sentence, output is Chinese sentence. Before the generating Chinese sentence , it must generate '39'. I generated 30 sentences. There was 29 sentences started with '39'. Of course, there is no sentence beginning with 39 in the English input. The generating effect is bad.

![up is original,down is generated](https://user-images.githubusercontent.com/20497531/103390440-5ece2580-4b4f-11eb-8162-b82e7c482792.png)
up is original,down is generated

I find my Chinese corpus, I only find one sentence which start with '39'. 
What caused this problem?
Should I use mt5 to do translation task?There is any good solution for translation task?
Thanks!

**Comment by craffel** [comment]
We have not experimented with using mT5 for translation. We use a constant learning rate during fine-tuning, but otherwise your plots look fine. Is the one sentence that starts with 39. repeated many times over the course of training, or something? Otherwise, I don't know what the issue is since I don't have access to your data, and this is not a use-case of mT5 that we intend to support.

**Comment by yayaQAQ** [comment]
> We have not experimented with using mT5 for translation. We use a constant learning rate during fine-tuning, but otherwise your plots look fine. Is the one sentence that starts with 39. repeated many times over the course of training, or something? Otherwise, I don't know what the issue is since I don't have access to your data, and this is not a use-case of mT5 that we intend to support.

There is only one sentence beginning with '39'. But generating results are all start with '39'. 
And what is the value of the constant learning rate? Maybe I can try it on my task.

**Comment by craffel** [comment]
1e-3, see the paper: 
<img width="667" alt="image" src="https://user-images.githubusercontent.com/417568/103415454-2549ea00-4b50-11eb-9dae-4aeab1cc6d4e.png">

**Comment by yayaQAQ** [comment]
@craffel Thanks!
